libraries/LittleFS/src/LittleFS.cpp

```diff
@@ -181,6 +181,12 @@ int LittleFSImpl::lfs_flash_sync(const struct lfs_config *c) {
 
 }; // namespace
 
+extern "C" void littlefs_weak_end(void)
+{
+    //ets_printf("debug: not weak littlefs end\n");
+    LittleFS.end();
```

Comment:
what happens when FS_NO_GLOBALS is defined? NO GLOBAL_INSTANCES? etc
what happens when FS_NO_GLOBALS etc  is defined and there is a manual instance of a FS?

Reply:
I think it would make sense to have the call to `end()` placed in the destructor.

Reply:
@devyte thanks. override is now only enabled with global instance is defined.
@TD-er littlefs already has a destructor, spiffs now calls `end()` on destructor